app/controllers/book_suggestion_controller.rb

```diff
@@ -0,0 +1,9 @@
+class BookSuggestionController < ApiController
+  def create
+    @book_suggestion = BookSuggestion.create(title: params[:book_suggestion][:title],
```

Comment:
Here you should add some validations for the params.

Reply:
Done